goaway_test.go

```diff
@@ -564,6 +594,10 @@ func TestFalseNegatives(t *testing.T) {
 			name:              "With Custom Dictionary",
 			profanityDetector: NewProfanityDetector().WithCustomDictionary(DefaultProfanities, DefaultFalsePositives, DefaultFalseNegatives),
 		},
+		{
+			name:              "With Custom Dictionary",
+			profanityDetector: NewProfanityDetector().WithCustomDictionary(DefaultProfanities, DefaultFalsePositives, DefaultFalseNegatives),
+		},
```

Comment:
```suggestion
		{
			name:              "With Custom Dictionary",
			profanityDetector: NewProfanityDetector().WithCustomDictionary(DefaultProfanities, DefaultFalsePositives, DefaultFalseNegatives),
		},
```
Was this a typo? It's a duplicate of the scenario above